index.html

```diff
@@ -1416,6 +1416,15 @@ <h3>Types</h3>
               </td>
             </tr>
 
+            <tr>
+              <td>
+<a href="#confirmation-method">confirmationMethod</a>&nbsp;object
```

Comment:
As mentioned during the F2F, `confirmationMethod` is the disadvantage of requiring significant context to understand. The thinking is that "confirmation" is too open ended and has many different uses in English -- and people without any of the context we had at the F2F could misuse it (accidentally?) in many ways.

Using the term `assurance` felt like a better fit, though people wanted another qualifier on that work. There were some objections to using "identity" as the qualifier: `identityAssurance`.

So, at the very least, I think `assuranceMethod` (without `identity` as a qualifier for now) would be an improvement over `confirmationMethod`.

```suggestion
<a href="#assurance-method">assuranceMethod</a>&nbsp;object
```

Reply:
a term `assurance` seems appropriate. but it needs to come with information about "assurance of what?" - just `assurance` is too general to be useful.
`holderBindingAssuranceMethod`?

Reply:
I'm partial to `identityAssuranceMethod` myself ... but I think @awoie objected to it, perhaps because of it's use in industry? However, it seems like there's actually some consonance where it could be properly used in conjunction with how it's already used in industry. I'm not sure if they could come around to the term :).

I do think many people have objected to the use of any language using the term "holder binding" generally, myself included. It's problematic for all the reasons we've discussed on the various issues related to it.

I think we should ask: "What is it that we want to get some level of assurance of?" ... and see how people respond here -- and if there is consensus on that. Hopefully that will lead to an acceptable qualifier.

Reply:
I like `assuranceMethod`. I wonder whether enough context can be supplied by where it's found, to address the wishes for `holderBindingAssuranceMethod` or `identityAssuranceMethod` or whatever? This would also allow it to be used in other scenarios and situations, not just involving verifiable credentials...

Reply:
I don't think `assurance` is better, I think it confuses `evidence` with `confirmation / holder binding`.

Reply:
I think assurance is worse in terms of required context to understand, especially given how varied it is used across digital identity technologies today. I personally like holderConfirmationMethod because it combines what the property is about ,"confirmation", with who is being confirmed, "the holder"

Reply:
`holderConfirmationMethod` is definitely an improvement over `confirmationMethod` (the latter of which I'd object to on its own), however, I'm still concerned that it's not quite right. Would `presenterConfirmationMethod` be better because it refers to whomever is presenting vs. anyone who may have held the VC at some time?

I still think "confirmation" is the wrong term; I feel it sends a false expectation:

A: "Hey, this person wasn't who you thought they were, I thought you confirmed this!"
B: "I did, I used the confirmation method!" <-- false expectation
A: "Well, it must have been hacked or shared with someone else. So I guess you didn't really confirm it was them after all."

Versus:

A: "Hey, this person wasn't who you thought they were, I thought you confirmed this!"
B: "Well, it can't be confirmed (without a doubt). I was reasonably sure it was them because I used the appropriate assurance method. That's the best I could do."
A: "Understood."

Additionally, I have a slight concern about unnecessarily limiting the use cases by pinning this to holder/presenter, or creating a pattern where we need a new feature for every new use case.

My understanding is that, in general, we want to be able to get some kind of identity assurance around what identifiers are expected to reference (i.e., that the verifier believes some identifier used in a VC refers to the same thing that the issuer believes it refers to). This is more broad than just getting assurance around the presenter or some particular holder at some point in time. Notably, this latter "point of time" variance around the 'holder' gives me pause for using that term as a qualifier in the name at all here... perhaps "presenterAssuranceMethod" instead?

But, maybe the types of methods used to provide assurance (again, not really "confirmation" which you can't do!) would only be limited to use "during presentation, by the presenter (or 'triggered / orchestrated' by the presenter)"? If the presenter is just triggering / orchestrating some assurance method, however, they are just driving the process for generating assurance that, for example, some IoT device does likely match against some identifier -- it isn't assurance that the presenter likely matches against some identifier.

Reply:
I think "confirmation" is the right word:
 
- https://www.rfc-editor.org/rfc/rfc7800.html#section-3

If we don't use the word "confirmation" we should at least link to the IETF concept, since it is essentially the same use case.

> This is analogous to the SAML 2.0 SubjectConfirmation element in which a number of different subject confirmation methods can be included (including proof-of-possession key information).

Reply:
Assurance can quickly be mixed with assuranceLevels and other places assurance have been used in identity. So we operate with 4 assuranceLevels in the nordics and it might get mixed directly to that meaning.

Reply:
@vongohren,

"Confirmation" is also problematic when understood from other contexts. For example, we do not want people to think that using a `confirmationMethod` means anything about the relative veracity of the claims of the VC:

> Confirmations are written responses from independent third parties verifying the accuracy of information requested by an auditor. They are used to test periodic account balances, such as cash, accounts receivable, consigned inventory, investment securities, notes payable, and lines of credit. Confirmations are presumed to provide greater assurance than examining source documents, as evidence acquired from independent sources outside the organization is generally deemed more reliable than evidence obtained solely from within the organization. Applying audit confirmation procedures appropriately can help auditors detect fraud early, which can limit damage to the organization and its shareholders.

[Source](https://go.gale.com/ps/i.do?id=GALE%7CA286972426&sid=googleScholar&v=2.1&it=r&linkaccess=abs&issn=00205745&p=AONE&sw=w&userGroupName=anon%7E7983eec)

I also don't think we should be using language from what are essentially authorization token frameworks (and which are thus contextualized around authorization) as our basis here. Both the third party model and the fact that VCs have much wider targeted use cases means we should be more explicit with what's going on and what the expectations are.

This is one reason why `holderConfirmationMethod` is a bit better -- as it at least pins it to something (the holder), even if you aren't really "confirming the holder", i.e., it's still not quite right. It's also the case that prepending "holder" seems like it will create problems with generalized identifier binding mechanisms and a possible proliferation of different prefixes.

Reply:
If we don't like "assurance" nor do we want to prefix it -- another word to consider working with is "confidence" ... as it implies a level of trust as opposed to knowledge (which "confirmation" erroneously implies).

Reply:
My understanding of what we want here is to enable an issuer to suggest methods that a verifier could use (or request to be used) to increase the verifier's ***confidence*** that:

1. A particular identifier refers to the same entity the issuer intended it to refer to.
2. (Optionally, the "holder binding case") That entity (or one of its delegates) is the presenter of the VC.

Reply:
It seems that assurance is not preferred by most people and confirmation without proper prefix does not have consensus either.

Proposals on the table are... 

1. `holderConfirmationMethod` (added the prefix)
2. `holderConfidenceMethod` (proposed by @dlongley )
3. `confidenceMethod` (see above but without prefix)

Note: ISO 23220 (mdoc) seems to have pivoted towards the term "holder verification", or "holder confirmation binding". Also note that in ISO there is defacto only holder == subject although they want to support other cases as well.

@dlongley @TallTed  would you be happy with option 1? If not, would you be happy with option 2., or 3.?

Reply:
@awoie,

Of the given choices, I'm happiest with this order (from most happy to least):

1. `confidenceMethod`
2. `holderConfidenceMethod`
3. `holderConfirmationMethod`

I'd strongly prefer NOT to do choice 3, `holderConfirmationMethod`, (maybe an objection if others agree) because it has these flaws:

1. "Confirmation" erroneously implies a level of knowledge vs. a level of trust (the latter is what this is for).
2. "Confirmation" has been used in a variety of different ways in industry, some that are similar and some that lead to mismatches / confusion (see above).
3. The particular prefix choice seems like it will encourage a proliferation of different terms for other "identifier binding" use cases that aren't specific to the holder.
4. The "holder" perhaps isn't even relevant for the "holder binding" use case, but rather the "presenter (or their delegate)" seems to be.

So, I don't like "holder" as a prefix, but *a prefix* helps solve some confusion when using "confirmation" ... but it doesn't solve other problems and introduces new ones.

I don't know if the non-prefixed `confidenceMethod` (my first choice here) avoids all of these things, but maybe it does.

Reply:
I'm generally with @dlongley, at this point.

I strongly disagree with continuing to refer to *anything* as "holder binding" because those words are just not anything we can do, and even using them colloquially can give the wrong impression to readers who are not deeply involved in our work.

I do not think "confirmation" is appropriate, because it involves a kind of proving that is not part of anything we've produced (or are likely, if able, to produce).

"Confidence" or "confidence method" seem OK to me. These might be combined with a "confidence level" to express the issuer's (or asserter's) faith in the "confidence method" and the specific assertion(s) to which this confidence applies.

Reply:
Yep, I agree that `holderBinding` would have been not appropriate since it cannot be guaranteed. However, to be clear, `confidenceMethod` could be a verification key, a DID, or anything else that a verifier may use to get more confidence that a transaction is made from the right entity.

I will then update the PR to use `confidenceMethod` instead of `confirmationMethod` if nobody objects: @OR13 @tplooker @paulbastian ?

Reply:
I don't like "confidenceMethod", I like "confirmationMethod"... it also aligned with https://www.rfc-editor.org/rfc/rfc7800.html#section-3

Reply:
@OR13, by "aligned", you mean NOT aligned? :P ... there can be only one of those in a `cnf` claim AND it is specific to proof-of-possession whereas this *is not*. There are other problems in industry related to the usage of the term in light of the veracity of claims and so on. I've also mentioned that I don't think our basis for language choice here should be what comes from essentially authorization token frameworks (and which are thus contextualized around authorization). VCs aren't the same thing by many miles.

Reply:
@awoie,

> Yep, I agree that `holderBinding` would have been not appropriate since it cannot be guaranteed. However, to be clear, `confidenceMethod` could be a verification key, a DID, or anything else that a verifier may use to get more confidence that a transaction is made from the right entity.

Yes, `confidenceMethod` covers both verification key / DID style methods and anything else (not restricted to proof-of-possession). I think that's the right read.

Reply:
_There's a gordian (*cough*) tangle between #942, #1054, #821, #902, and maybe others. I don't think that any of these can be resolved on its own._

_This may warrant a special topic call. Though, it might still be too soon for that._

According to RFC 7800, JWT `cnf` is about *proof* methods, primarily "proof-of-possession", with which to confirm "the authenticity of the token", which I think remain different from what we've been talking about for VCs, whether the final VC-related property name ends up as `holderBinding`, `confirmationMethod`, `confidenceMethod`, or something else.

Reply:
We have agreed on the special topic call (2nd of May) to use the term `confidenceMethod` and adjust the language in the PR to make it clear that the IETF definition is included as a subset of potential confidence methods.

Reply:
I'll update the PR according to the outcome of the special topic call from May 2nd.

Reply:
This conversation is resolved, also thanks to @dlongley and @TallTed concrete editorial suggestions.